bakery/src/scripts/assemble-book-metadata.py

```diff
@@ -1,18 +1,48 @@
 import sys
 import json
+from lxml import etree
+from pathlib import Path
 from cnxepub.collation import reconstitute
 from cnxepub.models import flatten_to_documents
+from cnxepub.html_parsers import DocumentMetadataParser
 
-in_path, out_path = sys.argv[1:3]
+ASSEMBLED_FILENAME = 'collection.assembled.xhtml'
 
-json_data = {}
+def get_module_xhtml_metadata(module_xhtml):
+    """Parse module metadata using XHTML file"""
+    with open(module_xhtml, "r") as module_xhtml_file:
+        html = etree.parse(module_xhtml_file)
+        metadata = DocumentMetadataParser(html)
 
-with open(in_path, "r") as in_file:
-    binder = reconstitute(in_file)
+    return metadata
 
-for doc in flatten_to_documents(binder):
-    abstract = doc.metadata.get("summary")
-    json_data[doc.ident_hash] = { "abstract": abstract }
+def main():
+    in_dir = Path(sys.argv[1]).resolve(strict=True)
+    output_file_path = sys.argv[2]
 
-with open(out_path, "w") as out_file:
-    json.dump(json_data, out_file)
+    input_assembled_file = in_dir / ASSEMBLED_FILENAME
+
+    json_data = {}
+
+    with open(input_assembled_file, "r") as in_file:
+        binder = reconstitute(in_file)
+
+    for doc in flatten_to_documents(binder):
+        # Parse the metadata from the supporting module XHTML files generated
+        # by neb during assemble to gain access to metadata that is in the
+        # head and doesn't get incorporated into the colllection assembled
+        # XHTML
+        module_xhtml_metadata = get_module_xhtml_metadata(
+            in_dir / f"{doc.id}.xhtml"
```

Comment:
This seems to be examining individual modules. Two things seem odd to me:
1. This seems like this must wait for assembly to finish.
2. It relies on the assembled-book input, which is not the source of these files

I'm not sure this is a blocker, but I thought I would pick your brain about those.

Reply:
woops, didn't send this in in time!

Reply:
It's never too late =). Happy to chat about the approach here to see if there's a better way, but probably easier to walk through and talk live. Want to hop on a call to discuss later this AM?

As far as the question for input, yes, it does assume assembly, but I'm not sure I understand the comment on the [assembled-book input](https://github.com/openstax/output-producer-service/blob/fa7d2af7ede80f71486cf53d8425e8fff9b629e8/bakery/src/tasks/assemble-book-metadata.js#L22) not being the source of the files.